### PR TITLE
+github.com/jpsim/SourceKitten

### PR DESCRIPTION
**Pull request by and-ri**

### CODE REVIEW

File: projects/github.com/jpsim/SourceKitten/package.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  ref: ${{version.tag}}
+
+versions:
+  github: jpsim/SourceKitten
+
+platforms:
+  - darwin
+
+build:
+  - patch -p1 < props/patch.diff
```
Comment (mxcl) [inline projects/github.com/jpsim/SourceKitten/package.yml]: patches require comments justifying why they are necessary.

**Review by mxcl — CHANGES_REQUESTED** [review]

File: projects/github.com/jpsim/SourceKitten/package.yml (hunk 1 above)
Comment (and-ri) [reply projects/github.com/jpsim/SourceKitten/package.yml]: I've added this comment and restricted the patch to version binding 👌🏻 

```
# https://github.com/jpsim/SourceKitten/pull/794 remove in release > 0.34.1
# fix this issue https://github.com/jpsim/SourceKitten/issues/793
- run: patch -p1 < props/patch.diff
  if: <=0.34.1
```

**Review by mxcl — APPROVED** [review]

**Comment by mxcl** [comment]
Depends on a newer swift that macos 11 provides https://github.com/pkgxdev/pantry/actions/runs/7918947298/job/21618717961

Not sure the intricacies of this yet. That is: do we actually need this to depend on macOS 12 or can we provide a swift version that is newer?

**Comment by and-ri** [comment]
> Depends on a newer swift that macos 11 provides https://github.com/pkgxdev/pantry/actions/runs/7918947298/job/21618717961
> 
> Not sure the intricacies of this yet. That is: do we actually need this to depend on macOS 12 or can we provide a swift version that is newer?

```
SourceKitten now requires Swift 5.6 or higher to build, and macOS 12
or higher to run.
```

from: https://github.com/jpsim/SourceKitten/releases/tag/0.33.0

**Comment by mxcl** [comment]
k well we need to be able to specify these platform dependencies.

**Pull request merged**